models/modules/img2img_turbo/img2img_turbo.py

```diff
@@ -200,9 +201,15 @@ def forward(self, x, prompt):
             return_tensors="pt",
         ).input_ids.cuda()
         caption_enc = self.text_encoder(caption_tokens)[0]
-
         # match batch size
-        captions_enc = caption_enc.repeat(x.shape[0], 1, 1)
+        batch_size = caption_enc.shape[0]
```

Comment:
Unneeded ?

Reply:
inside cut_model, x is created by cat real_A and real_B, which double the batch size of the x tensor.  prompt tensor has normal batch_size, so, to match the two tensor, I did this modification. Detail toy example is here:   https://colab.research.google.com/drive/1RMvHt2PuQufH4zEc2Lrds561L9NEYzYf?usp=sharing

Reply:
This should be fixed by modifying the prompt tensor outside turbo, in cut when A & B are concatenated for inference, not here.